app/server.ts

```diff
@@ -0,0 +1,41 @@
+import * as Express from 'express';
+import * as BodyParser from 'body-parser';
+import * as Morgan from 'morgan';
+
+import { PORT } from './modules/util/config';
+import { BasePath as ProjectBasePath, projectRouter } from './modules/project';
+
+export class Server {
+    server: Express.Application;
+    port: string;
+
+    constructor() {
+        this.server = Express();
+        this.port = PORT;
+        this.config();
+        this.defaultRoute();
+    }
+
+    config = () => {
```

Comment:
This method is more of a `attachMiddlewares` sort of method, calling it `config` could be misleading. What do you think ?

- [ ] Can `config` be renamed to something meaningful ?